templates/cli/lib/commands/push.js.twig

```diff
@@ -658,7 +726,7 @@ const generateChangesObject = (attribute, collection, isAdding) => {
     return {
         key: `${chalk.yellow(attribute.key)} in ${collection.name} (${collection['$id']})`,
         attribute: attribute,
-        reason: isAdding ? 'Field doesn\'t exist on the remote server' : 'Field doesn\'t exist in appwrite.json file',
+        reason: isAdding ? 'Field isn\'t available on the remote server' : 'Field isn\'t available on the appwrite.json file',
```

Comment:
```suggestion
        reason: isAdding ? 'Field isn\'t present on the remote server' : 'Field isn\'t present on the appwrite.json file',
```

Isn't available still sounds like warning a little.
Isnt present sounds like a cause.

This might be just me. If you are not onboard, we can ask Dennis to pick